lib/attribrutex/model.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Attribrutex.Model do
+  defmacro __using__([]) do
+    quote do
+      def unquote(:custom_field_changeset)(params) do
```

Comment:
The `unquote(:custom_field_changeset)` is not needed.
This would be the same as

```elixir
quote do
  def custom_field_changeset(params) do
    params = Map.put(params, :fieldable_type, module_to_string(__MODULE__))
    Attribrutex.CustomField.changeset(%Attribrutex.CustomField{}, params)
  end
end
```